fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/provider/impl/FileProvider.java

```diff
@@ -41,9 +44,10 @@
 
     private static final Logger logger = Logger.getLogger(FileProvider.class.getName());
 
+    private static final long READ_BLOCK_OPT = 524288L;
```

Comment:
Any rationale for picking this size? If so, might be worth a comment

Reply:
Yes, I tried this with a few difference sizes.  Let me add an implNote